custom/icds_reports/templates/icds_reports/mobile/dashboard/mobile_dashboard.html

```diff
@@ -0,0 +1,88 @@
+{% load menu_tags %}
```

Comment:
copied and stripped down from the web version. I'm still unclear whether this should be a totally different page, whether they should extend the same base template, or whether they should be the same template with everything handled in context variables. but for now I wanted something standalone and separate so i could see the minimal set of stuff required to render a basic program summary.